Application/StorageManager/Interfaces/IStorageManager.cs

```diff
@@ -0,0 +1,52 @@
+using Domain.Search.ShortlistTuitionPartners;
+
+namespace Application.StorageManager.Interfaces;
+
+public interface IStorageManager
```

Comment:
`IStorageManager` is a pretty generic name. Maybe this should be `ITuitionPartnerShortlistStorage` or something?

Reply:
Yes, you are right. Kieran and I had talked about renaming it. I will do it soonest👍